docs/developing/install.rst

```diff
@@ -77,11 +76,11 @@ Install the following packages:
 .. note:: Unfortunately you need to install the ``postgresql`` package, because
           Homebrew does not currently provide a standalone ``libpq`` package.
 
-Upgrade pip and install tox:
+Upgrade pip and install tox and tox-pip-extensions:
 
 .. code-block:: bash
 
-    pip install -U pip tox
+    pip install -U pip tox tox-pip-extensions
```

Comment:
This command does not specify the tox version, so the user will end up with the incompatible tox 3.5.0 by default.

Reply:
Fixed